docs/supported-data-types.md

```diff
@@ -1,23 +1,25 @@
-# Target Source files
+# Supported Input Formats
+
+This code generator supports the following input formats:
+
 - OpenAPI 3 (YAML/JSON, [OpenAPI Data Type](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#data-types))
 - JSON Schema ([JSON Schema Core](http://json-schema.org/draft/2019-09/json-schema-validation.html) /[JSON Schema Validation](http://json-schema.org/draft/2019-09/json-schema-validation.html))
 - JSON/YAML Data (it will be converted to JSON Schema)
 - Python dictionary (it will be converted to JSON Schema)
 
-# Implemented list
+## Implemented data types and features
 
-This codegen supports major data types to OpenAPI/JSON Schema
+Below are the data types and features recognized by datamodel-code-generator for OpenAPI 3 and JSON Schema.
 
-## OpenAPI 3 and JsonSchema
-### DataType
-- string (include patter/minLength/maxLenght)
-- number (include maximum/exclusiveMaximum/minimum/exclusiveMinimum/multipleOf/le/ge)
-- integer (include maximum/exclusiveMaximum/minimum/exclusiveMinimum/multipleOf/le/ge)
```

Comment:
These were removed as I could not find any reference to them in the OpenAPI or JSON Schema docs